jack-sys/build.rs

```diff
@@ -128,12 +128,17 @@ fn write_dynamic_loading_src<W: std::io::Write>(out: &mut W) -> Result<(), std::
                 f.type_name()
             )?;
         } else {
+            let mut f_name = f.name;
```

Comment:
This probably needs to be locked behind the dynamic loading feature as dynamic linking (write_dynamic_linking_src function) doesn't work due to having the same symbol name ("jack_client_open") and Rust not supporting varargs (https://github.com/rust-lang/rust/issues/44930)